lifelines/fitters/aalen_johansen_fitter.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import print_function
+from __future__ import division
+import numpy as np
+import pandas as pd
+
+from lifelines.fitters import KaplanMeierFitter
+from lifelines.utils import _preprocess_inputs
+
+
+class AalenJohansenFitter():
+    """
+    Class for fitting the Aalen-Johansen estimate for the cumulative incidence function in a competing 
+    risks framework.
+    
+    AalenJohansenFitter()
+    
+    """
+    def jitter(event_time,jitter_level=0.00001,seed=None):
```

Comment:
this is an internal function, so we should prepend the function name with with `_`